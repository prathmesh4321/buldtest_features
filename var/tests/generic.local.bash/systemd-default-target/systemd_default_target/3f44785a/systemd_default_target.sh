#!/usr/bin/bash 
# Content of run section
if [ "multi-user.target" == `systemctl get-default` ]; then
  echo "multi-user is the default target";
  exit 0
fi
echo "multi-user is not the default target";
exit 1