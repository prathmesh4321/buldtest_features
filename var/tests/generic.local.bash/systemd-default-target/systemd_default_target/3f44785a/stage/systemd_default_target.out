multi-user is not the default target
